samples/f5.py

```
@@ -0,0 +1,116 @@
+import cv2
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)